### PR TITLE
Add ADIOS2

### PR DESCRIPTION
**Pull request by ax3l**
Add support for ADIOS2 backends and transports in ParaView.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -112,6 +113,7 @@ requirements:
     - python
     - numpy
     - matplotlib-base  # [unix]
+    - openpmd-api
```
Comment (ax3l) [inline recipe/meta.yaml:116]: Let's do in a later PR, resolver seems very confused.
```suggestion
```

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Comment by jschueller** [comment]
you probably need https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10113

**Comment by jschueller** [comment]
needs an update, feel free to reopen

**Pull request closed without merging**